pkg/controller/baremetalhost/baremetalhost_controller.go

```diff
@@ -385,6 +386,39 @@ func (r *ReconcileBareMetalHost) Reconcile(request reconcile.Request) (reconcile
 		}
 	}
 
+	// Check the current power status against the desired power
+	// status.
+	if provResult, err = prov.UpdateHardwareState(); err != nil {
```

Comment:
Is this something that accesses the BMC, and thus needs to be throttled to ~once a minute? Is something below this call doing throttling? I didn't see it, but maybe it's happening in ironic or somewhere else I didn't notice.

Reply:
The Provisioner is responsible for returning a result with a delay value (see line 399 for where it is used). In the current implementation, it's just a database read because Ironic is polling the BMC on its own schedule.

Reply:
Perfect. I did see the delay from line 399, but wanted to make sure we weren't depending on the controller's queue to guarantee that it won't reconcile sooner.